.github/pull_request_template.md

```diff
@@ -17,10 +17,8 @@ _Put any questions or notes for the reviewer here._
 - [ ] Did you write tests?
 - [ ] Did you write documentation?
 - [ ] Did you make any API changes? Update the corresponding API documentation.
-- [ ] If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️
 
 # Reviewer Checklist
-* Comment with /azp run test-porter-release if a magefile or build script was modified
 * Comment with /azp run porter-integration if it's a non-trivial PR
```

Comment:
Sorry I wasn't clear, we should just remove the entire Reviewer checklist. There isn't a "azp run porter-integration" support on the operator repo at all. It's only a thing for the core repo.